.github/workflows/changelog-assembly.yml

```diff
@@ -21,9 +32,15 @@ jobs:
           crate: aeruginous
 
       - run: |
-          aeruginous ronlog release \
-            -i changelog.d/ \
-            -v "$(cargo r -- --version | cut -d\  -f2)"
+          aeruginous increment-version \
+            -v "$(cat .version)" \
+            -r ${{ inputs.release }} \
+            -e .version \
+            -e CITATION.cff \
+            -e Cargo.lock \
+            -e Cargo.toml
```

Comment:
At first, we increment the version numbers in all important files.  If there should be more in the future, just add them to this list.

Reply:
I tend to keep the version in one place only and use cargo work-spaces when I have different crates in the same project (Which I don't use here anymore since crates.io forces that every crate must be published there too)

So this should always work, unless something major happens here (Like adding a REST api Backend)